rules/windows/builtin/win_net_use_admin_share.yml

```diff
@@ -0,0 +1,26 @@
+title: Mounted Windows Admin Shares with net.exe
+id: 3abd6094-7027-475f-9630-8ab9be7b9725
+status: experimental
+description: Detects when an admin share is mounted using net.exe
+references:
+    - https://drive.google.com/file/d/1lKya3_mLnR3UQuCoiYruO3qgu052_iS_/view
+author: Teymur Kheirkhabarov '@HeirhabarovT', Zach Stanford '@svch0st'
+date: 2020/10/05
+tags:
+    - attack.lateral_movement
+    - attack.T1021.002
```

Comment:
Could you please also update the attack technique tag? It supposed to be [lowercase](https://github.com/Neo23x0/sigma/wiki/Rule-Creation-Guide#tags). There was a CI test that was checking for this kind of typos (it happened quite often), but most probably it was changed/disabled due to sub-techniques release.